packages/appeals-service-api/src/routes/v2/users/_id/appeal-submissions/_appealSubmissionId/controller.js

```diff
@@ -0,0 +1,30 @@
+const { getAppealSubmissionForUser } = require('./service');
+const logger = require('#lib/logger');
+const ApiError = require('#errors/apiError');
+
+/**
+ * @type {import('express').Handler}
+ * @param {import('express').Request<{ id: string, appealSubmissionId: string }>} req
+ */
+async function getUserAppealSubmission(req, res) {
+	const { id: userId, appealSubmissionId } = req.params;
+	try {
+		const content = await getAppealSubmissionForUser({ userId, appealSubmissionId });
+		if (!content) {
+			throw ApiError.userNotFound();
+		}
+		res.status(200).send(content);
+	} catch (error) {
+		if (error instanceof ApiError) {
+			logger.error(`Failed to get users: ${error.code} // ${error.message.errors}`);
+			res.status(error.code || 500).send(error.message.errors);
```

Comment:
You can just throw here, common middleware handles ApiError - but no harm having it explicit.

Reply:
Oh, this was a copy paste so I'll change both to throw

Reply:
changed this and the one I copied from :D